### DISCUSSION TITLE
Placeholder API

### DISCUSSION THREAD

**Discussion by Ugz12** [PlaceholderAPI/PlaceholderAPI#1023] [(2) Support]
Every time I add the placeholderAPI plugin to my plugins it does not detect the expansions of other plugins such as Vault, Luck Perms and others.
"But you can use the command /papi ecloud download <expansion>"
It works for plugins that are in the cloud, but any Scoreboard plugin I use doesn't work.
it literally doesn't work, someone help me, I'm using the SimpleScore Plugin

**Comment by Andre601** [comment]
The plugins you mentiond don't include the expansions by themself, meaning you have to install them using the PAPI ecloud command.

Also, it's up to the scoreboard plugin to actually provide PlaceholderAPI support. If the plugin doesn't add it, then there is nothing we can do.
Double check that the plugin actually supports PlaceholderAPI and that you actually use the right placeholder patterns.

**Reply by Ugz12** [reply]
What are you saying by "if the plugin was added" why was it supposed to be added when installing SimpleScore?

**Reply by Andre601** [reply]
I never said that. Read again.
